src/components/PublicHeader.spec.tsx

```diff
@@ -30,4 +31,20 @@ describe('PublicHeader', () => {
         mount(<PublicHeader user={ dummyUser }/>);
         cy.get('[data-test="user-avatar"]').should('be.visible');
     });
+
+    it('contains a zetkin logo when component has no org prop', () => {
+        mount(<PublicHeader user={ dummyUser }/>);
+        cy.get('[data-test="zetkin-logotype"]').should('be.visible');
+    });
+
+    it('contains org avatar instead of zetkin logo when component has an org prop', () => {
+        cy.fixture('dummyOrg.json')
+            .then((data : ZetkinOrganization) => {
+                const dummyOrg = data;
+                mount(<PublicHeader org={ dummyOrg } user={ dummyUser }/>);
```

Comment:
The reassignment to a new `dummyOrg` variable is unnecessary. Pass the `data` argument directly to the component, and optionally rename it to `dummyOrg` if you prefer that name.